nextflow.config

```diff
@@ -30,10 +30,27 @@ profiles {
     params.validation_script = "dynamic_challenge_validate.py"
     params.email_script = "dynamic_challenge_send_email.py"
   }
-  pegs_challenge {
-    params.view_id = "syn53770151"
-    params.input_id = "syn51390589"
-    params.scoring_script = "model_to_data_score.py"
+  pegs_challenge_validate {
+    params.project_name = "PEGS DREAM Challenge - staging"
```

Comment:
Can you remind me again, what this parameter is used for?  I wanted to see whether we should use the name of the staging site or the prod one.

Reply:
`project_name` --> used to get the `synid` of the project --> which is used to get the `synid` of the root `Logs/` folder. This is used to create folders + update permissions in `create_folders.py`, and then used again to update said folders in `update_folders.py`:

* Here is where it gets called in [create_folders.py](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/blob/dffa9497f1eb6316f3acdfcd5d19e52638967021/bin/create_folders.py#L118)
* And in [update_folders.py](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/blob/dffa9497f1eb6316f3acdfcd5d19e52638967021/bin/update_folders.py#L82)

Reply:
Ah then we should probably use `PEGS DREAM Challenge` in that case, since the staging site is only accessible to organizers.  Unless, @gaiaandreoletti, we need to hide the Docker logs from the participants?